.github/workflows/rust.yml

```
@@ -14,150 +16,71 @@ env:
   CARGO_UNSTABLE_SPARSE_REGISTRY: true
 
 jobs:
-  msrv:
-    name: MSRV
+  build:
+    name: Check on ${{ matrix.rust }}
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - 1.61.0
```

```suggestion
          - 1.61.0 # MSRV
```
Could you add a comment indicating MSRV? This is because I guess we can forget why this version is evaluated.